src/components/sortableList/SortableListItem.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, {PropsWithChildren, useCallback, useContext} from 'react';
+import {LayoutChangeEvent} from 'react-native';
+import {
+  useSharedValue,
+  useAnimatedReaction,
+  withTiming,
+  Easing,
+  runOnJS,
+  useAnimatedStyle,
+  withSpring
+} from 'react-native-reanimated';
+import {GestureDetector, GestureUpdateEvent, PanGestureHandlerEventPayload} from 'react-native-gesture-handler';
+import View from '../view';
+import SortableListContext from './SortableListContext';
+import usePresenter from './usePresenter';
+import useDragAfterLongPressGesture from './useDragAfterLongPressGesture';
+
+interface SortableListItemProps {
+  index: number;
+}
+
+type Props = PropsWithChildren<SortableListItemProps>;
+
+const animationConfig = {
+  easing: Easing.inOut(Easing.ease),
+  duration: 350
+};
+
+const SortableListItem = (props: Props) => {
+  const {children, index} = props;
+
+  const {itemHeight, itemsOrder, onChange} = useContext(SortableListContext);
+  const {getTranslationByIndexChange, getItemIndexById, getIndexByPosition, getIdByItemIndex} = usePresenter();
+  const translateY = useSharedValue(0);
+  const tempTranslateY = useSharedValue(0);
+  const tempItemsOrder = useSharedValue(itemsOrder.value);
+
+  const onLayout = useCallback((event: LayoutChangeEvent) => {
```

Comment:
I noticed that onLayout is being called twice with different heights (minor difference)

Reply:
I did not see that, are you sure you did not have two devices at the time? If you still see this, let's look at it together.

Reply:
Yes, you might be right.